i18n/en.json

```diff
@@ -3239,6 +3243,190 @@
     "id": "api.templates.deactivate_subject",
     "translation": "[{{ .SiteName }}] Your account at {{ .ServerURL }} has been deactivated"
   },
+  {
+    "id": "api.templates.delinquency_14.button",
+    "translation": "Update payment"
+  },
+  {
+    "id": "api.templates.delinquency_14.subject",
+    "translation": "Payment is overdue for your Mattermost {{.Plan}}."
+  },
+  {
+    "id": "api.templates.delinquency_14.subtitle1",
+    "translation": "We weren't able to charge the credit card we have on file. This means your workspace is at risk of being downgraded to Cloud Starter."
```

Comment:
Based on what we talked about today referencing `Cloud Starter` in our team planning, @johndavidlugtu do you also want to remove it from here?

Reply:
Sorry, I missed the ping here.

We can keep it. Because this is delinquency emails (have knowledge of their paid plan), we can assume they know Cloud Starter.

@marianunez